### DISCUSSION TITLE
Add support for plotting

### DISCUSSION THREAD

**Discussion by martazvch** [lennart-finke/GodoTeX#10] [Ideas]
Hi, after playing around with the LaTeX expressions (which work great !), I was wondering if it would be possible to render function plotting? If it is, I'de love to participate but I don't know where to start.

**Comment by lennart-finke** [comment]
Hello again @MartinVacheron. I'm glad the rendering works!

Plotting is outside the scope of this project. If you would like to work on this, maybe you can help out in a different add-on closer to what you had in mind, for example [here](https://github.com/fenix-hub/godot-engine.easy-charts), or start your own repository. All the best in your coding ventures, in any case!
